.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -41,7 +41,7 @@ The ARM reviewer on-call engineer visits the merge queue twice a day, so the app
 What's the purpose of this PR? Check the specific option that applies. This is **mandatory**!
 
   - [ ] New resource provider.
-  - [ ] New API version for an existing resource provider. (If API spec is not defined in TypeSpec, the PR should have been generated using [OpenAPI Hub](https://aka.ms/openapihub)).
+  - [ ] New API version for an existing resource provider. (If API spec is not defined in TypeSpec, the PR should have been generated using [OpenAPI specs automation](https://aka.ms/azsdkdocs/createopenapispec)).
```

Comment:
For all the other aka.ms links I followed the convention of `aka.ms/azsdk/` not `aka.ms/azsdkdocs/`. I also see that page doesn't mention it has a short-link.

Reply:
Ohh oops. is there guidance on this so people know how to create the links? 
also, the link is present:
![image](https://github.com/Azure/azure-rest-api-specs/assets/9868623/0976f94a-1ce1-44e2-97b3-303c8d97de1b)

Reply:
I'm leaving the link as is and later we can change it and update accordingly